app/src/config.json

```diff
@@ -1,22 +1,22 @@
 {
     "signatureScheme": "Token-Ed25519-SHA512",
     "urls": {
-        "local": "http://localhost:8000",
-        "dev": "https://api.dev.token.io",
-        "stg": "https://api.stg.token.io",
-        "perf": "https://api.perf.token.io",
-        "sandbox": "https://api.sandbox.token.io",
-        "int": "https://api.int.token.io",
-        "prd": "https://api.token.io"
+        "local": "http://localhost:8000${TOKEN_PATH_PART}",
```

Comment:
We can't have `TOKEN_PATH_PART` here, this is a publicly facing SDK. The `TOKEN_PATH_PART` is only for internal services.
Instead, we should list a new environment `beta` in this config, i.e.
```
"beta": "https://api.token.io/beta"
```